src/components/register/register.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { ReactElement, useState } from 'react';
+
+import { CustomLink } from '../../utils/helpers';
+import './register.scss';
+
+interface IRegisterFormState {
+  readonly email: string;
+  readonly password: string;
+}
+
+export function Register(): ReactElement {
+  const [form, setState] = useState<IRegisterFormState>({
+    email: '',
+    password: '',
+  });
+
+  const updateField = (event: React.ChangeEvent<HTMLInputElement>): boolean => {
+    setState({
+      ...form,
+      [event.target.name]: event.target.value,
+    });
+
+    return false;
+  };
+
+  return (
+    <div className="c-register">
+      <h2 className="c-register__header">Register</h2>
+      <form className="c-register__form">
+        <input
```

Comment:
Would it be possible to add labels for accessibility purposes? I personally prefer them over the `placeholder`, as they don't disappear, but I guess it'd be OK to hide the labels with CSS if you'd rather, which would still allow screen readers to recognize the fields correctly.

If so, same goes for the login page.